### PR TITLE
Update readme, remove sigv4a crt section

### PR DESCRIPTION
**Pull request by alextwoods**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

1. To make sure we include your contribution in the release notes, please make sure to add description entry for your changes in the "unreleased changes" section of the `CHANGELOG.md` file (at corresponding gem). For the description entry, please make sure it lives in one line and starts with `Feature` or `Issue` in the correct format.

2. For generated code changes, please checkout below instructions first:
  https://github.com/aws/aws-sdk-ruby/blob/version-3/CONTRIBUTING.md

Thank you for your contribution!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,14 +274,9 @@ by adding the `aws-crt` gem to your Gemfile.
...
 [S3 Multi-Region Access Points](https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiRegionAccessPoints.html)
-and Amazon EventBridge Global Endpoints.
-Currently sigv4a requires the [aws-crt](https://rubygems.org/gems/aws-crt/) gem.  The `aws-sigv4` signers 
-will automatically use the CRT provided signers with support for `sigv4a` when the `aws-crt` gem is available.
-
-```ruby
-gem 'aws-sdk-s3', '~> 1'
-gem 'aws-sigv4', '1.4.1.crt'
-```
+and Amazon EventBridge Global Endpoints.  Sigv4a is supported natively in Ruby
```
Comment (mullermp) [inline README.md]: May want to rewrite this entire section about how CRT only speeds up sigv4a and mentioning what sigv4a is used for seems out of place and not necessary. It should be transparent to customers.

Comment (alextwoods) [reply README.md]: CRT sigv4a is currently slower with ffi bindings, so didn't want to mention it.  I can move the sigv4a section into its own section elsewhere in the README or just get rid of it.  Thoughts?

Comment (mullermp) [reply README.md:277]: Whatever you'd like, my comment was to raise awareness that this section may or may not even be necessary anymore.

Comment (mullermp) [reply README.md]: I think the sigv4a section in CRT can probably just be removed. We already say how to install it and why (crc32c)

**Review by mullermp — APPROVED** [review]

**Pull request merged**